h5py/tests/test_filters.py

```diff
@@ -61,6 +62,7 @@ def test_wr_scaleoffset_fletcher32(self):
                                   )
 
 
+@pytest.mark.mpi_break
```

Comment:
For clarity, I might suggest calling the marker `mpi_skip` or `mpi_xfail` - you might even allow both to distinguish "this shouldn't work" vs "this should work but doesn't".

Reply:
I've release a new version of pytest-mpi which adds `mpi_skip` and `mpi_fail`, which should be more understandable.